code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+##########################################################
+# Name of file: 04_Census_and_Bed_Days.R
+# Data Release: Delayed Discharges monthflagly publication
+# Original author(s): Peter McClurg (spss version: James Mc Nally)
+# Original date: 09/10/19 (spss version: 30/11/2017)
+# latest update author (if not using version control)
+# latest update date (if not using version control)
+# latest update description (if not using version control)
+# Type of script: Preparation
+# Written/run on: R Studio SERVER
+# Version of R that the script was most recently run on: ?
+# Description of content: Updates trendfile with DD records from most recent publication monthflag.
+# Approximate run time: TBC
+##########################################################
+ 
+
+### 1.Housekeeping ----
+# This section should be the only section of the script which requires manual changes 
+# for future updates and includes:
+#   loading package_grps
+#   setting filepaths and extract dates
+#   functions (defined here or sourced from another file)
+#   setting plot parameter
+#   specifying codes (e.g. ICD-10 codes)
+
+
+
+### Filepaths for latest monthflag # may NOT NEEDED as now within set-up_environment ( check with Russell)
+
+filepath<-("/conf/delayed_discharges/RAP development/2020_02/Outputs/")
+filepath2<-("/conf/delayed_discharges/RAP development/2020_02/Data/scotland/")
+
+### 2. Bring in environment ----
+#( ran separately in code folder - how do i bring it in without copying / pasting as ?)
+source("00_setup_environment.R")
+
+#add in new library needed that isn't in setup.
+library(xlsx)
+
+# monthflagflag<-("Apr 2020")
+
+### 3. Get Scotland_validated file for latest monthflag ----
+
+datafile<-read.csv(paste0(filepath2,"SCOTLAND_validated.csv"))
```

Comment:
This file doesn't exist and isn't created from previous scripts? Assuming it's the output from `02_create_scotland_file.R`, the method to read in that file without using a hardcoded file path is suggested below.
```suggestion
datafile <- readRDS(here("data", format(start_month, "%Y-%m"), paste0(format(start_month, "%Y-%m"), "_scotland.rds")))
```